SU-SOE/soe_basic
# D8CORE-3499- fixing the arrows on the cards for publications

# READY FOR REVIEW

# Summary
- In the release prep we lost this change in a merge. So after consulting with @imonroe , I made a new branch off master and fixed the issue. 
- On the publication node, the cards at the bottom should have the SOE arrow and the external link. On the hover the only change is that the blue underline changes to black. 
- Would it be possible to see this fix on SOE Test for verifying?

# Review By (Date)
- ASAP

# Urgency
- HIGH

# Steps to Test

1. This is a branch from master. So make sure your local jives. 
2. Pull in this change and refresh.
3. Add or make sure you have external links publications and local ones. You will need 4.
4. Look at the Card in the More Publications on a node page. Notice how the hovers work. There should be the SOE arrow for internal links and the arrow for external.  There should be a blue underline and the text and icons black.  The only change on hover should be the blue underline changing to black. 

# Affected Projects or Products
- SOE
- Publications

# Associated Issues and/or People
- D8CORE-3499 fixup from merging losses? We think that is what happened
- @imonroe @pookmish 

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
